pyaedt/generic/general_methods.py

```diff
@@ -100,18 +101,29 @@ def _exception(ex_info, func, args, kwargs, message="Type Error"):
     _write_mes("https://aedtdocs.pyansys.com/search.html?q={}".format(func.__name__))
     _write_mes("************************************************************")
 
-
-def _get_remote_args(arg):
+def _check_types(arg):
     if "netref.builtins.list" in str(type(arg)):
+        return "list"
+    elif "netref.builtins.dict" in str(type(arg)):
+        return "dict"
+    elif "netref.__builtin__.list" in str(type(arg)):
+        return "list"
+    elif "netref.__builtin__.dict" in str(type(arg)):
+        return "dict"
+    else:
+        return ""
```

Comment:
Saves a line, you can ignore this.

```suggestion
    return ""
```